visivo/models/selector.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum
+from pydantic import ConfigDict, Field, PrivateAttr, model_serializer
+
+from visivo.models.base.base_model import BaseModel
+from visivo.models.base.named_model import NamedModel
+
+
+class SelectorType(str, Enum):
+    single = "single"
```

Comment:
What's the default selection if you choose single? I don't know if it makes sense to do in this pass but we should allow people to specify a default selection(s) for both single and multi. I guess single would be one string and multi would be a list of strings maybe?

Reply:
It grabs the "first" one.  Yea I agree with that!